### ISSUE TITLE
how does tus name checksums?

### ISSUE THREAD

**Issue by nigoroll** [tus/tus-resumable-upload-protocol#156]
Ref https://github.com/tus/tus-resumable-upload-protocol/blob/master/protocol.md#checksum

This question might sound picky, but would it not make sense for tus to define how checksums are named or refer to a 3rd party for that definition?
A clarification has already been added that names of hashes are in lower case, yet for example it is unclear to me if SHA3-224 would be called `sha3_224`, `sha3-224` or even differently.

**Comment by Acconut** [comment]
Good point! There are no strict rules since this question has never arisen but in your particular case, I would stick to the rule of making it lowercase. So SHA3-224 becomes `sha3-224`.

**Comment by nigoroll** [comment]
sure, that would work. But I wonder if the project should have a simple registry (need not be more than a file with hash name identifiers)?

**Comment by Acconut** [comment]
I don't think we need a complete registry where we are required to keep up with the latest hashing algorithms. But maybe we can add a few more naming examples to the spec, that cover most naming conventions. What do you think?

**Comment by nigoroll** [comment]
I still think that a simple registry would leave the least amount of slack at very low overhead.

**Comment by Acconut** [comment]
Can you provide a few examples which would "leave room for interpretation"? Please don't take this as an offense, I am just not sure what variety of naming we have to deal with.

**Comment by nigoroll** [comment]
room for interpretation examples:  `sha3-256` vs. `sha-3-256` vs `sha3_256` vs. `sha_3_256`  etc.

**Comment by Acconut** [comment]
If you feel so strongly about this, please open a PR with your idea so we can understand what your concrete proposal is like.
